src/features/health/agent.controller.js

```diff
@@ -0,0 +1,106 @@
+const { Agent } = require('../../shared/db/mongodb/schemas');
+
+const createAgent = async (req, res) => {
+    const { first_name, last_name, email, region } = req.body;
+
+    //check if req fields provided
+    if (!first_name || !last_name || !email || !region) {
+        return res.status(400).send({
+            success: false,
+            message: 'first_name, last_name, email and region are required'
+        });
+    }
+
+    //Set sales to 0 and default values
+    const agent = await Agent.create ({
+        first_name,
+        last_name,
+        email,
+        region,
+        sales: 0, // makes sales default to 0
```

Comment:
Your schema already includes a default value for sales as is right? If so, this is probably redundant.

Reply:
Yes, its in the schema. I'll take this one out.